apps/src/templates/teacherDashboard/SectionActionDropdown.jsx

```diff
@@ -0,0 +1,66 @@
+import React, {Component, PropTypes} from 'react';
+import color from "../../util/color";
+import {sortableSectionShape} from "./shapes.jsx";
+import {OverlayTrigger} from 'react-bootstrap';
+import SectionActionBox from "./SectionActionBox";
+
+const styles = {
+  actionButton:{
```

Comment:
Please add cursor:pointer to the chevron button. and maybe a hover style on the button; it doesn't feel super discoverable right now.